pyw3d/objects.py

```diff
@@ -820,16 +913,25 @@ def apply_material(self, blender_object):
             not self["double_sided"])
         color = [channel/255.0 for channel in self["color"]]
         color.append(int(self["visible"]))
+        print(color)
         blender_object.color = color
+        
+        if self["lighting"] is True:
```

Comment:
I think we discussed this before (and discovered that I was wrong), but I'll risk embarrassment one more time. This check shouldn't be necessary, right? Whether or not an object is lit shouldn't affect whether or not it receives a color, right?